ntbs-service/Pages/Admin/Migration.cshtml

```diff
@@ -1,105 +1,104 @@
+                                is-error-input=@notificationDateStartError asp-for="NotificationDateRangeStart.Year" />
+                        </nhs-form-group>
+                    </nhs-date-input-item>
+                </nhs-date-input>
+            </nhs-form-group>
+        </div>
+
+        <div>
+            @{
+                var notificationDateEndError = !Model.ValidationService.IsValid("NotificationDateRangeEnd");
+                var notificationDateEndGroupState = notificationDateEndError ? Error : Standard;
+            }
+            <nhs-form-group nhs-form-group-type=@notificationDateEndGroupState classes="nhs-form-group--import-by-date">
+                <span nhs-span-type="ErrorMessage" id="notification-end-date-error" classes="import-error"
+                    asp-validation-for="NotificationDateRangeEnd" has-error="@notificationDateEndError"></span>
+                <label nhs-label-type="Standard" for="notification-end-date"> End Notification Date Range </label>
+                <nhs-date-input id="notification-date">
+                    <nhs-date-input-item>
+                        <nhs-form-group nhs-form-group-type="Standard">
+                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Day">Day</label>
+                            <input nhs-input-type="Date" fixed-width="Two"
+                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Day" />
+                        </nhs-form-group>
+                    </nhs-date-input-item>
+                    <nhs-date-input-item>
+                        <nhs-form-group nhs-form-group-type="Standard">
+                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Month">Month</label>
+                            <input nhs-input-type="Date" fixed-width="Two"
+                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Month" />
+                        </nhs-form-group>
+                    </nhs-date-input-item>
+                    <nhs-date-input-item>
+                        <nhs-form-group nhs-form-group-type="Standard">
+                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Year">Year</label>
+                            <input nhs-input-type="Date" fixed-width="Four"
+                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Year" />
+                        </nhs-form-group>
+                    </nhs-date-input-item>
+                </nhs-date-input>
+            </nhs-form-group>
+        </div>
+
+        <div>
+            <button nhs-button-type="Standard" type="submit" value="Import" classes="ntbsuk-button--primary submit-search-button" id="search-button">
+                Import
+            </button>
+        </div>
+    </form>
+
+    <br/>
+    <a href='/Hangfire/jobs/succeeded'> Navigate to dashboard </a>
```

Comment:
It feels like this shouldn't navigate to Hangfire... but happy to be ignored if there is logic to this.

Reply:
oh fair! I think this was requested originally when the connection between them was quite un-clear!